README.future.md

```diff
@@ -18,16 +18,83 @@ const client = await Pinecone.createClient({
 
 It also can find information specified in the environment variables `PINECONE_API_KEY` and `PINECONE_ENVIRONMENT`.
 
-## Data Plane
+## Control Plane
+
+### Create an index
+
+The minimum required configuration to create an index is the index name and dimension. The dimension you choose should match the output of the model used to produce your embeddings.
+
+```
+await client.createIndex({ name: 'my-index', dimension: 128 })
+```
+
+In a more expansive example, you can specify the metric, number of pods, number of replicas, and pod type.
+
+```
+await client.createIndex({
+ name: 'my-index',
+ dimension: 128,
+ metric: 'cosine',
+ pods: 1,
+ replicas: 2,
+ podType: 'p1.x1'
+})
+```
+
+By default all metadata fields are indexed when vectors are upserted with metadata, but if you want to improve performance you can specify the specific fields you want to index. This example is showing a few hypothetical metadata fields, but the values you'd use depend on what metadata you plan to store in Pinecone alongside your vectors.
+
+```
+await client.createIndex({ name: 'my-index', dimension: 128, metadataConfig: { 'indexed' : ['productName', 'productDescription'] }})
```

Comment:
just fyi (don't think we should change anything here at this point) -- this probably won't be a thing for v4, but itll be silently ignored in the api for backwards compatability see https://github.com/pinecone-io/pinecone-db/pull/1556

Reply:
This is good context. I think a lot of things will definitely need to be adjusted for v4. I should start a list.